tests/test_pipeline.py

```diff
@@ -0,0 +1,54 @@
+from ac_dc.badwords import badwords
+from ac_dc.config import BaseConfig
+from ac_dc.basic_pipeline import BasicPipeline
+from tqdm import tqdm
+import gzip
+from tqdm import tqdm as tqdm
+
+
+def test_basic_pipeline():
+    lang_oscar_id = "en"
+    path_model_fasttext = "/tmp/lid.176.bin"
+    path_oscar_file = "/tmp/en_part_1.txt.gz"
+
+    parameters_filtering_en = {
+        "cond_remove_words_with_incorrect_substrings": True,
+        "incorrect_word_substrings": ["http", "www", ".com", "href", "//"],
+        "cond_remove_long_words": True,
+        "length_word_cutoff": 25,
+        "cond_check_empty": True,
+        "strip_characters": "' 0123456789¯_%$§½¼¾×|†~\"—±′–'°−{}[]·'?,./<>!@#^&*()+-‑=:;，：,...�`→¶.…‘’”",
+        "cond_check_special_characters": True,
+        "special_characters": "' 0123456789¯_%$§½¼¾×|†~\"—±′–'°−{}[]·'?,./<>!@#^&*()+-‑=:;，：,...�`→¶.…‘’”",
+        "special_characters_cutoff": 0.4,
+        "cond_check_stopwords": True,
+        "stopwords_cutoff": 0.4,
+        "cond_check_badwords": True,
+        "badwords_cutoff": 0.4,
+        "badwords": badwords[lang_oscar_id],
+        "cond_check_lang_id": True,
+        "path_model_fasttext": path_model_fasttext,
+        "lang_oscar_id": lang_oscar_id,
+        "lang_id_cutoff": 0.8,
+    }
+
+    config = BaseConfig("en", **parameters_filtering_en)
```

Comment:
how would we load a language specific config? Do we inherit from `BaseConfig` and then have a dict mapping per language?

```
{
   "en": EnglishConfig,
   "vi": VietnameseConfig,
}
```

Reply:
I also felt that `config` + `pipeline` now is quite cumbersome and not a good decision to make it.